frontend/src/app/trees/forests/tree-guidelines/tree-guidelines.component.html

```diff
@@ -1,2 +1,53 @@
+<section id="sidebar-page" class="usa-section usa-content">
+  <div class="usa-grid breadcrumbs" *ngIf="forest">
+    <a routerLink="/christmas-trees/forests">Christmas tree permits</a> > <span><a routerLink="/christmas-trees/forests">Find a forest</a></span><span> > {{ forest.forestName }} National Forest{{ forest.forestName.indexOf('and') > 0 ? 's' : ''}} Christmas tree permit information</span>
+  </div>
+  <div class="usa-grid">
+    <app-sidebar mobileMenu="true" [items]="sidebarItems"  class="usa-width-one-third usa-layout-docs-sidenav"></app-sidebar>
 
-<app-sidebar-view *ngIf="template === 'sidebar'" [forest]="forest"></app-sidebar-view>
+    <div class="usa-width-two-thirds sidebar-page-main-content">
+      <h1 *ngIf="forest">{{forest.forestName}} National Forest{{ forest.forestName.indexOf('and') > 0 ? 's' : ''}} Christmas Tree Permit Information</h1>
+      <br/>
+      <p *ngIf="forest">Christmas trees can be cut anywhere on the {{forest.forestName}} National Forest subject to the rules and regulations on your permit and as noted below.</p>
+
+
+      <app-section-heading heading="When to cut your tree" elementId="cutting-dates"></app-section-heading>
+      <app-tree-cutting-dates [forest]="forest"></app-tree-cutting-dates>
+      <br>
+      <button *ngIf="forest" [routerLink]="['/applications/christmas-trees/forests/', forest.forestAbbr,'new']" class="usa-button-primary-alt" id="buy-permit-link">Buy permit</button>
+      <hr class="thick">
+      <app-section-heading heading="How to choose your tree" elementId="tree-selection"></app-section-heading>
+      <app-tree-species [forest]="forest"></app-tree-species>
+
+      <hr class="thick" />
+      <app-section-heading heading="How to cut your tree" elementId="cutting-instructions"></app-section-heading>
+      <app-section-heading heading="Before you cut" elementId="cutting-instructions-before-you-cut" selector="h3"></app-section-heading>
+      <hr class="trees-section-divider">
+      <app-tree-tools [forest]="forest"></app-tree-tools>
+      <app-section-heading heading="When you cut" elementId="cutting-instructions-when-you-cut" selector="h3"></app-section-heading>
+      <hr class="trees-section-divider">
+      <app-tree-cutting-rules [forest]="forest"></app-tree-cutting-rules>
+
+      <hr class="thick" />
+      <app-section-heading heading="Rules to know" elementId="need-to-know"></app-section-heading>
```

Comment:
does it make sense to make a more generalizable "section component" that you can feed in a heading and then a content - so that there is more consistency across text layout pages?

Reply:
I think it would be good to create a section component that feeds a heading and you can define the component which stores the content. I've made a story for this: #221